sigv4-signing/java/SigV4Signer.java/README.md

```diff
@@ -0,0 +1,51 @@
+| uri | The URI to sign. The result from GetSignalingChannelEndpoint plus the relevant Query Parameters. For more information, see [ConnectAsMaster](https://docs.aws.amazon.com/kinesisvideostreams-webrtc-dg/latest/devguide/kvswebrtc-websocket-apis-2.html) or [ConnectAsViewer](https://docs.aws.amazon.com/kinesisvideostreams-webrtc-dg/latest/devguide/kvswebrtc-websocket-apis-1.html). | wss://v-a1b2c3d4.kinesisvideo.us-west-2.amazonaws.com?X-Amz-ChannelARN=arn:aws:kinesisvideo:us-west-2:123456789012:channel/demo-channel/1234567890123&X-Amz-ClientId=d7d1c6e2-9cb0-4d61-bea9-ecb3d3816557 |
+| wssUri | Secure WebSocket method "wss" plus hostname obtained from GetSignalingChannelEndpoint                                                                                                                                                                                                                                                                                                    | wss://v-a1b2c3d4.kinesisvideo.us-west-2.amazonaws.com                                                                                                                                                     |
+| accessKey | AWS Access Key                                                                                                                                                                                                                                                                                                                                                                           | AKIAIOSFODNN7EXAMPLE                                                                                                                                                                                      |
+| secretKey | AWS Secret Key                                                                                                                                                                                                                                                                                                                                                                           | wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY                                                                                                                                                                  |
+| sessionToken | AWS Session Token. If non-temporary credentials are used, should be an empty String.                                                                                                                                                                                                                                                                                                     | AQoDYXdzEJr...\<remainder of token\>
```

Comment:
| sessionToken | AWS Session Token. If non-temporary credentials are used, otherwise should be an empty String e.g. sessionToken := ""

Reply:
Updated